rusk/src/lib/chain/rusk.rs

```diff
@@ -503,10 +501,12 @@ fn execute(
     Ok(receipt)
 }
 
-fn update_hasher(hasher: &mut Sha3_256, event: Event) {
-    hasher.update(event.source.as_bytes());
-    hasher.update(event.topic.as_bytes());
-    hasher.update(event.data);
+fn update_hasher(hasher: &mut Sha3_256, events: &[Event]) {
```

Comment:
Not to be handled here, but is there any reason why this is SHA3 and not Blake2? I believe we use Blake2 everywhere and we probably should remain consistent

Reply:
Have no idea from where it's originated